core/src/main/scala/org/apache/gearpump/cluster/client/ClientContext.scala

```diff
@@ -20,8 +20,8 @@ package org.apache.gearpump.cluster.client
 
```

Comment:
ClientContext is quite low level, and has significant impact to almost all classes.

Can you try to build things on top of ClientContext, instead of changing it?